### DISCUSSION TITLE
Camunda 8 using Zeebe Spring java to get Tasklist via code

### DISCUSSION THREAD

**Discussion by arenishr** [camunda-community-hub/community#114] [Q&A]
Am a camunda 7 user, now we moved to camunda 8. 
Since c7 was easily integrated with java code, we defined our intermediate service for c8 is also java.
Tested the basic TAskHandlers, deployment, instance creation etc. But it seems no straight way to access tasklist via the code in java. Kindly put some insights or the best practice to do so.
Tried:
CamundaTaskListClient client = new CamundaTaskListClient.Builder().taskListUrl("http://localhost:8082").authentication(sa).build();
Http Approach to get token:
Request request = new Request.Builder()
				.url("http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token")
			    .post(formBody)
			    .header("Content-Type", "application/x-www-form-urlencoded")
			    .build();
Still the tasklist api if through REST, it needs the Session data (String cookie = response.headers().map().get("Set-Cookie").get(0).toString();).
Here am stuck in the flow..

**Comment by xomiamoore** [comment]
Hi @arenishr, please post your question in the [forums](https://forum.camunda.io/). That's the best way to get an answer as this GitHub community is for our community projects, plugins, and other tools.
